lib/view_models/page_view_model/profile_page_viewModel.dart

```diff
@@ -43,12 +43,12 @@ class ProfilePageViewModel extends BaseModel {
   bool get isCreator => _isCreator;
   AuthController get authController => _authController;
 
-  initialize(BuildContext context, bool isCreator, List<User> test) {
-    if (isCreator != null && test != null) {
-      _userDetails = test;
-      _isCreator = isCreator;
-      _org = _userDetails[0].joinedOrganizations;
-    }
+  initialize({BuildContext context}) {
```

Comment:
Hi @TheHazeEffect 
I will do it.

Thanks

Reply:
@TheHazeEffect 
Also, this dart file is showing Warnings because of the camel case in the file name. Should I change it to _profile_page_view_model.dart_?

Reply:
Sure, just place it in it's own commit, easier for me to review

Reply:
@TheHazeEffect I have renamed the file in another commit (eb3f815). 
Thanks